internal/database/mariadb/test/fixture.go

```diff
@@ -1209,11 +1211,24 @@ func NewFakeComponentInstance() mariadb.ComponentInstanceRow {
 	}
 }
 
+var nextUserType entity.UserType = entity.CasualUserType
+
+func getNextUserType() int64 {
+	userType := nextUserType
+	if userType == entity.CasualUserType {
+		nextUserType = entity.TechnicalUserType
+	} else {
+		nextUserType = entity.CasualUserType
```

Comment:
Maybe we use `EndUserType` or `HumanUserType` instead of `CasualUserType`. Otherwise it might give the impression that the technical user is a human user with technical knowledge, but it's supposed to be like a service account that is used for automation

Reply:
Done